### PR TITLE
unit tests for vasp details components

### PR DESCRIPTION
**Pull request by elysee15**

### CODE REVIEW

File: web/gds-admin-ui/src/pages/app/details/TrixoQuestionnaire/index.js [hunk 1]
```diff
@@ -31,36 +22,38 @@ function TrixoForm({ data }) {
...
-                        <p><span className='fw-bold'>Minimum Compliance Threshold:</span> {intlFormatter({ currency: data?.compliance_threshold_currency }).format(data?.compliance_threshold)} {data?.compliance_threshold_currency}</p>
+                        <p><span className='fw-bold'>Minimum Compliance Threshold:</span> <span data-testid="compliance_threshold_currency">{intlFormatter({ currency: data?.compliance_threshold_currency }).format(data?.compliance_threshold)}</span> {data?.compliance_threshold_currency}</p>
                     </Col>
                     <Col xs={12} md={6}>
                         <h5 className='text-black'>CDD & Travel Rule Policies</h5>
                         <hr className='mt-1' />
-                        <p className='lh-lg'>
+                        <p className='lh-lg' data-testid="has-required-regulatory-program">
                             Organization <span className='fw-bold'>{hasRequiredRegulatoryProgram[data?.has_required_regulatory_program]}</span> programme that sets minimum AML, CFT, KYC/CDD and Sanctions standards per the requirements of the jurisdiction(s) regulatory regimes where it is licensed/approved/registered.
```
Comment (pdeziel) [inline web/gds-admin-ui/src/pages/app/details/TrixoQuestionnaire/index.js:54]:
```suggestion
                            Organization <span className='fw-bold'>{hasRequiredRegulatoryProgram[data?.has_required_regulatory_program]}</span> meet minimum AML, CFT, KYC/CDD and Sanctions standards per the requirements of the jurisdiction(s) regulatory regimes where it is licensed/approved/registered.
```

**Review by pdeziel — CHANGES_REQUESTED** [review]
Overall it seems like we have a good test pattern developed for validating the VASP detail fields, well done!

I did have several spelling and wording changes, mostly `organisation` -> `organization` and `juridictions` -> `jurisdictions` and also some wording changes in the TRIXO questionnaire. I definitely missed a few so please correct any of those I didn't tag in the comments, and rebuild and test to verify that the changes still work.

File: web/gds-admin-ui/src/pages/app/details/TrixoQuestionnaire/index.js (hunk 1 above)
Comment (elysee15) [reply web/gds-admin-ui/src/pages/app/details/TrixoQuestionnaire/index.js:54]: `Organization has a partially implemented programme that sets minimum AML` What will happen if we have a sentence like this ?

Comment (pdeziel) [reply web/gds-admin-ui/src/pages/app/details/TrixoQuestionnaire/index.js:54]: This wording makes sense to me, can we change `programme` to `program` to be consistent with the other US english spelling? This actually brings up a really good point that we haven't officially made a decision of which spelling to prefer (US english vs. British english), but for right now let's stick to US english spelling of words to be consistent with the rest of the UI.

File: web/gds-admin-ui/src/pages/app/details/TrixoQuestionnaire/index.js [hunk 2]
```diff
@@ -31,36 +22,38 @@ function TrixoForm({ data }) {
...
                     </Col>
                     <Col xs={12} md={6}>
                         <h5 className='text-black'>CDD & Travel Rule Policies</h5>
                         <hr className='mt-1' />
-                        <p className='lh-lg'>
-                            Organization <span className='fw-bold'>{hasRequiredRegulatoryProgram[data?.has_required_regulatory_program]}</span> programme that sets minimum AML, CFT, KYC/CDD and Sanctions standards per the requirements of the jurisdiction(s) regulatory regimes where it is licensed/approved/registered.
+                        <p className='lh-lg' data-testid="has-required-regulatory-program">
+                            Organization <span className='fw-bold'>{hasRequiredRegulatoryProgram[data?.has_required_regulatory_program]}</span> program that sets minimum AML, CFT, KYC/CDD and Sanctions standards per the requirements of the jurisdiction(s) regulatory regimes where it is licensed/approved/registered.
```
Comment (pdeziel) [inline web/gds-admin-ui/src/pages/app/details/TrixoQuestionnaire/index.js:54]: This is showing up as `Organization does program that sets minimum...`, can we change the wording to `Organization has program that sets minimum...`?

**Review by pdeziel — APPROVED** [review]
@elysee15 I think this is almost there, just one more suggested change and then we can merge this in.

**Pull request merged**